src/exchanges/mutateBody.ts

```diff
@@ -0,0 +1,159 @@
+import { pipe, tap, map } from 'wonka';
+import { Exchange, Operation, OperationResult } from '../types';
+import {
+  DocumentNode,
+  print,
+  ASTNode,
+  buildClientSchema,
+  visitWithTypeInfo,
+  TypeInfo,
+  FragmentDefinitionNode,
+} from 'graphql';
+import { parse, visit, NonNullTypeNode } from 'graphql';
+import { type } from 'os';
+import { object } from 'prop-types';
```

Comment:
This is unintended I assume

Reply:
Yeah it's in a messy state - not ready for review/merge yet :)

Reply:
Prefixed it with `WIP` so that noone finds this and assumes it's about to be imminently merged in :)